src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml

```diff
@@ -2015,8 +2016,8 @@ let print_version_info () = Core.printf "Commit %s\n" Mina_version.commit_id
 
 let () =
   Random.self_init () ;
-  let compile_config = Mina_compile_config.Compiled.t in
-  let logger = Logger.create ~itn_features:compile_config.itn_features () in
+  let itn_features = Sys.getenv "ITN_FEATURES" |> Option.is_some in
```

Comment:
@dkijania I understand that you've put this value to env variables because you wanted to have the value before parsing the CLI arguments.

Still, I suggest that we read it from the runtime config. And read itn-related CLI arguments independently of `itn_features` value. Just that if `itn_features: false` after reading the config, Mina node will crash if itn-related flag or command was provided.

Reply:
Understand, but what to due with rest of commands that not necessarily expose runtime_config arg like advanced itn-create-accounts